.github/workflows/db-charm-tests.yaml

```diff
@@ -11,20 +11,32 @@ jobs:
         charm-repo:
           - "canonical/postgresql-operator"
           - "canonical/postgresql-k8s-operator"
-# TODO: uncomment once we've fixed https://github.com/canonical/operator/issues/987
-#          - "canonical/mysql-operator"
-#          - "canonical/mysql-k8s-operator"
+          - "canonical/mysql-operator"
+          - "canonical/mysql-k8s-operator"
 
     steps:
       - name: Checkout the ${{ matrix.charm-repo }} repository
         uses: actions/checkout@v3
         with:
           repository: ${{ matrix.charm-repo }}
 
+      - name: Checkout the operator repository
+        uses: actions/checkout@v3
+        with:
+          path: myops
```

Comment:
Are we 100% sure this will checkout the changes from this PR (specifically, the temp merge of this PR branch into `main`) and not the latest on `main`?

Reply:
> # The branch, tag or SHA to checkout. When checking out the repository that
> # triggered a workflow, this defaults to the reference or SHA for that event.
> # Otherwise, uses the default branch.

[source](https://github.com/actions/checkout)

Reply:
Thanks!